README.md

```diff
@@ -1,7 +1,248 @@
+
+
+#### Terminology
+| Term                            | Description                                                                                                                                     |
+|---------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------|
+| CSM                             | crosslink spectra match                                                                                                                         |
+| PSM                             | peptide (or crosslink) spectra match                                                                                                            |
+| peptide pair                    | 2 peptides crosslinked to each other, no matter the crosslink site. Multiple CSMs can support the same peptide pair                             |
+| Link/crosslink/residue pair     | 2 residues crosslinked to each other. Multiple peptide pairs can support the same crosslink                                                     |
+| Protein pair/PPI                | 2 proteins found crosslinked to each other, no matter the number/identity of the residue pairs. Multiple residue pairs can support the same PPI |
+| linear peptide                  | a non-crosslinked peptide                                                                                                                       |
+| Self link                       | a link within a protein sequence (includes links between multiple copies of the same protein)                                                   |
+| Heteromeric link                | a link between 2 different protein sequences                                                                                                    |
+| Ambiguity                       | when peptide pairs or residue pairs can be mapped to multiple proteins because these share the same sequence                                    |
+| Protein Group                   | the group of proteins sharing the sequence making up the ambiguous group                                                                        |
+| Prefilter                       | a score filter applied prior to FDR calculation to both target and decoy matches                                                                |
+| Local FDR                       | the estimation of FDR of a particular match based on a windowed approach or a fitting of the score distributions.                               |
+| Posterior Error Probability/PEP | the square root of the local FDR                                                                                                                |
```

Comment:
Where did you find that? To my understanding the terms local FDR and PEP are basically interchangeable

Reply:
in the tooltip if you hover over the "local FDR" box check

Reply:
that is a bit of an misunderstanding. `square` is one of three states of the checkbox. It means when the checkbox is filled (guess I should call it that way) then it calculates PEP/local FDR. If it is ticked, then PEP will be calculated and filtered on, meaning xiFDR will first filter to e.g. 5% FDR and then to a local FDR or PEP of 5%. If it is empty then the PEP is not calculated to begin with.

Reply:
i see- yes i understood what the checkbox does and i documented it in the table. But then why do we use the words PEP at all? I will change this in a commit

Reply:
History for that: our lab, a long time ago,  made a paper (https://doi.org/10.1186/1471-2105-10-179) that was talking about local FDR. In other software, e.g. MaxQuant this is revert to PEP. So people might have heard of one or the other - but I am not sure what is the best name. In a way I calculate that is very close to our original paper. So I was sticking with that name in the interface. But I have PEP in the tooltip for people who have heard of that term but not local FDR.

Reply:
I see- in the new commit there is one line that says "Local FDR/PEP" so that it is clear